mtgjson5/providers/tcgplayer.py

```diff
@@ -24,6 +24,20 @@ class TCGPlayerProvider(AbstractProvider):
     """
 
     api_version: str = ""
+    condition_map = {1: "NM", 2: "LP", 3: "MP", 4: "HP", 5: "DMG", 6: "U"}
```

Comment:
I'd like to see these values built at run time instead of static, just in case they change something on their API. 
See https://docs.tcgplayer.com/reference#catalog_getconditions should get condition map. Language map, I'm not sure at this exact moment.